lib/modules/login/view/login_screen.dart

```diff
@@ -1,17 +1,53 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:prepowl/modules/dashboard/view/dashboard.dart';
 
+import '../controller/login_bloc.dart';
 import 'login_screen_ui/login_top_image.dart';
 import 'login_screen_ui/mobile_login_form.dart';
 import 'social_screen_ui/socal_sign_up.dart';
 
+class Login extends StatelessWidget {
+  const Login({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => LoginBloc(),
+      child: const LoginPageScreen(),
+    );
+  }
+}
+
 class LoginPageScreen extends StatelessWidget {
   const LoginPageScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: SingleChildScrollView(
-        child: MobileSignupScreen(),
+    return Scaffold(
+      body: BlocBuilder<LoginBloc, LoginState>(
```

Comment:
Use BlocConsumer and put logic of navigating to dashbaord screen here... check if **isAuthenticated** then move to dashboard. dont put dashboard ui here